src/components/ClassicElasticTab.tsx

```diff
@@ -16,7 +16,7 @@ import { MouseoverTooltip } from './Tooltip'
 
 function ClassicElasticTab() {
   const qs = useParsedQueryString()
-  const tab = (qs.tab as string) || VERSION.ELASTIC
+  const tab = qs.tab && typeof qs.tab === 'string' && qs.tab in VERSION ? (qs.tab as VERSION) : VERSION.ELASTIC
```

Comment:
qs.tab in VERSION` this condition always `false`

Reply:
Good catch Danh. It will work if the key and value of each pair in the enum are equally like this
```ts
enum VERSION {
  ELASTIC = 'ELASTIC',
  CLASSIC = 'CLASSIC',
}
```

Maybe we can use Object.values here

Reply:
Fixed. Playground to test: [link](https://www.typescriptlang.org/play?#code/KYOwrgtgBAagogJQMoEkDyA5KBvAUFKOAGQEEkAVFAYSgF4oByYAGwEMBnAFwEsBjBgDT4oVUklQ16DXm3bs+g3AF9cuXgHsQXKN3YoQccNHoAeclGAAPTqAAm7KAmAaATrZNcX3EAHMBUT28fAD5ggApPAC4Azi9ff1BIAAVWF1YIaPIASmjPHQdyAG0Aa2AAT3UAMyhyAF06YJxhF2BOMBcQKDQAIwArZ04AOgA3VmYwYHYwxIgUtIgswe8ZMFtJiNis5VUNLU4oVlzYoLpGGQ55fh3NdnVmYEHmdR8w3X1DSDDWf3hkdAwsltuNVXnoDEYvgJfqhMICmgQCKxlEA)

Reply:
`qs.tab &&` typeof qs.tab === 'string' && qs.tab in VERSION ...

can we remove qs.tab here

Reply:
I prefer keep it